### PR TITLE
fix title. Closes #180

### LINKED ISSUE #180: title not being shown?
This is weird... https://rc.glamorous.rocks/

screen shot 2017-06-30 at 12 57 24 pm

Anyone know what's going on?

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: Removes `NextScript` from `_document`

**Why**: Fixes #180

**How**: Remove it. Nothing appears broken 🤷‍♂️

### CODE REVIEW

File: pages/_document/index.js [hunk 1]
```diff
@@ -25,10 +25,11 @@ export default class MyDocument extends Document {
     return (
       <html lang={this.props.locale}>
         <Head>
-          <title>{content.title}</title>
```
Comment (kentcdodds) [inline pages/_document/index.js]: Sometimes prettier-atom does weird things 🤷‍♂️

File: pages/_document/index.js [hunk 2]
```diff
@@ -25,10 +25,11 @@ export default class MyDocument extends Document {
...
+          <title>
+            {content.title}
+          </title>
           <style dangerouslySetInnerHTML={{__html: this.props.css}} />
           <meta name="viewport" content="width=device-width, initial-scale=1" />
-          {/*  */}
```
Comment (kentcdodds) [inline pages/_document/index.js]: not sure what this was here for 🤔

File: pages/_document/index.js [hunk 3]
```diff
@@ -37,7 +38,6 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           <Main />
-          <NextScript />
```
Comment (kentcdodds) [inline pages/_document/index.js]: I have no idea what this is for, but nothing seems to break when I remove it and it fixes the title issue so I think we're ok.

Comment (paulmolluzzo) [inline pages/_document/index.js]: LOL. I thought that injects the client side bundle. It doesn't?

Comment (paulmolluzzo) [inline pages/_document/index.js]: Wondering if a static export has anything to do with it. On mobile so I can't investigate.

Comment (kentcdodds) [inline pages/_document/index.js]: Haha, yeah, I misunderstood you.

And when I said "nothing seems to break" I mean: "I was a bad developer and everything looked fine but nothing actually functioned." lol

Comment (paulmolluzzo) [inline pages/_document/index.js]: LOL. It happens.

**Pull request merged**